src/index.html

```diff
@@ -4,9 +4,385 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img 
+        src="./images/logo.png" 
+        alt="logo"
+        >
```

Comment:
```suggestion
        <img 
          src="./images/logo.png" 
          alt="logo"
        >
```
Check yourself and fix in all cases

Reply:
done